### PR TITLE
[report-converter] Support Golint

### PR DESCRIPTION
**Pull request by csordasmarton**

### CODE REVIEW

**Comment by gyorb** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- tools/report-converter/codechecker_report_converter/golint/output_parser.py  3
- tools/report-converter/codechecker_report_converter/golint/analyzer_result.py  2
- tools/report-converter/tests/unit/test_golint_parser.py  3
- tools/report-converter/codechecker_report_converter/golint/plist_converter.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/CodeChecker/codechecker/pullRequest?prid=5210268&bid=17085738)

**Comment by gyorb** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- tools/report-converter/tests/unit/test_golint_parser.py  3
- tools/report-converter/codechecker_report_converter/golint/analyzer_result.py  2
- tools/report-converter/codechecker_report_converter/golint/plist_converter.py  1
- tools/report-converter/codechecker_report_converter/golint/output_parser.py  3
         

Clones added
============
- tools/report-converter/tests/unit/test_golint_parser.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/CodeChecker/codechecker/pullRequest?prid=5210268&bid=17085738)

**Comment by gyorb** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- tools/report-converter/codechecker_report_converter/golint/output_parser.py  3
- tools/report-converter/codechecker_report_converter/golint/analyzer_result.py  2
- tools/report-converter/tests/unit/test_golint_parser.py  3
         

Clones added
============
- tools/report-converter/tests/unit/test_golint_parser.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/CodeChecker/codechecker/pullRequest?prid=5210268&bid=17085738)

**Comment by gyorb** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- tools/report-converter/codechecker_report_converter/golint/analyzer_result.py  2
- tools/report-converter/tests/unit/test_golint_parser.py  3
- tools/report-converter/codechecker_report_converter/golint/output_parser.py  3
         

Clones added
============
- tools/report-converter/tests/unit/test_golint_parser.py  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/CodeChecker/codechecker/pullRequest?prid=5210268&bid=17085738)

File: tools/report-converter/codechecker_report_converter/analyzer_result.py [hunk 1]
```diff
@@ -72,6 +72,14 @@ def _add_report_hash(self, plist_objs):
 
     def _get_analyzer_result_file_content(self, result_file):
```
Comment (bruntib) [inline tools/report-converter/codechecker_report_converter/analyzer_result.py:73]: I had some comments on this function in #2649.

Comment (csordasmarton) [reply tools/report-converter/codechecker_report_converter/analyzer_result.py:73]: @bruntib I will check it and fix your comments on that pull request.

**Pull request merged**